modules/ocl/cl_context.cpp

```diff
@@ -441,10 +457,15 @@ CLContext::create_va_image (const cl_libva_image &image_info)
 {
     cl_mem mem_id = NULL;
     cl_int errcode = CL_SUCCESS;
-    if (!is_valid())
+    cl_platform_id platform_id = _device->get_platform_id ();
+    if (!is_valid() || !platform_id)
         return NULL;
 
-    mem_id = clCreateImageFromLibvaIntel (_context_id, &image_info, &errcode);
+    OCLCREATEIMAGEFROMLIBVAINTEL *oclCreateImageFromLibvaIntel =
+        (OCLCREATEIMAGEFROMLIBVAINTEL *) clGetExtensionFunctionAddressForPlatform (platform_id, "clCreateImageFromLibvaIntel");
```

Comment:
OCLCREATEIMAGEFROMLIBVAINTEL defined for function pointer. no need to define another pointer.
just OCLCREATEIMAGEFROMLIBVAINTEL oclCreateImageFromLibvaIntel